clients/client/src/main/java/org/projectnessie/client/api/QueryBuilder.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.client.api;
+
+public interface QueryBuilder<R extends QueryBuilder<R>> {
+
+  /** Required: the <a href="https://github.com/projectnessie/cel-java">CEL</a> script. */
```

Comment:
`Required`? Should it not be optional, meaning get "all" if no query expression is given?

Reply:
This is just moving the existing code + comments.

Reply:
ok